Resources/Private/JavaScript/Host/Containers/LeftSideBar/index.js

```diff
@@ -10,17 +10,20 @@ import style from './style.css';
 const {$get} = immutableOperations;
 
 @connect(state => ({
-    isHidden: $get(state, 'ui.leftSideBar.isHidden')
+    isHidden: $get(state, 'ui.leftSideBar.isHidden'),
+    isFullScreen: $get(state, 'ui.fullScreen.isFullScreen')
 }))
 export default class LeftSideBar extends Component {
     static propTypes = {
-        isHidden: PropTypes.bool.isRequired
+        isHidden: PropTypes.bool.isRequired,
+        isFullScreen: PropTypes.bool.isRequired
     };
 
     render() {
         const classNames = mergeClassNames({
             [style.leftSideBar]: true,
-            [style['leftSideBar--isHidden']]: this.props.isHidden
+            [style['leftSideBar--isHidden']]: this.props.isHidden,
```

Comment:
Agreed. Quick question: Wouldn't it make sense to even put that into @connect in https://github.com/PackageFactory/PackageFactory.Guevara/pull/37/files#diff-7f5ed69c466f20c5f6463003e7164ed8L12 - or would it be better to let the container handle the fullscreen state separately?